fastlane/lib/fastlane/actions/docs/frame_screenshots.md

```diff
@@ -21,19 +21,30 @@ _frameit_ allows you to put a gorgeous device frame around your iOS and macOS sc
 
 # Features
 
+## Frame screenshot
+
 Put a gorgeous device frame around your iOS and macOS screenshots just by running one simple command. Support for:
 
 - iPhone, iPad and Mac
 - Portrait and Landscape modes
-- Several colors
+- Several device frame colors
 
 The complete and updated list of supported devices and colors can be found [here](https://github.com/fastlane/frameit-frames/tree/gh-pages/latest)
 
 Here is a nice gif, that shows _frameit_ in action:
 
 ![img/actions/FrameitGit.gif](/img/actions/FrameitGit.gif?raw=1)
 
-### Results
+## Advanced Features
+
+- Put framed screenshot on colored background, define padding
```

Comment:
Should `define padding` be on its own line? Maybe `- Define custom padding`?

Reply:
I only focussed on adding the missing parameters, not on these kind of optimisations. I prefer to leave this as is.